scripts/run_all_dex-net_4.0_pj_examples.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+echo "RUNNING EXAMPLE 0"
+python examples/policy.py --depth_image data/examples/clutter/primesense/depth_0.npy --segmask data/examples/clutter/primesense/segmask_0.png --config_filename cfg/examples/replication/dex-net_4.0_pj.yaml
```

Comment:
This should use example images from PhoXi. Let's collect some when we get a chance. Should be very quick

Reply:
Sure, I believe that there is a script tools/capture_images.py in BerkeleyAutomation/perception, but that won't give us segmasks, right? I guess I can manually extract them from the policy on the real robot if you tell me what specific images we want.

Reply:
Sounds good.